test/test_entity-manager.js

```diff
@@ -106,6 +106,66 @@ define(function (require) {
                 expect(manager.entityHasComponent(entity, 'Position')).to.be.true;
                 expect(manager.entityHasComponent(entity, 'Unit')).to.be.true;
             });
+
+            it('create event is raised from the manager to the processor', function () {
+                var manager = prepareManager();
+                var newEntityId = 42;
+
+                var processor = {
+                    entities: [],
+                    on(type, data) {
```

Comment:
Is this valid JS syntax? I don't think I've seen it before, so it's confusing me. I'd have expected `on: function(...) {`. Just like you did for `update` below. Even if it's valid, do you mind making it coherent and using an explicit `function` keyword?

Reply:
Yeah right. I think I kind of copied/pasted the code you put in #24 and used it as the test code.